datalake/src/main/java/com/sequenceiq/datalake/flow/dr/event/DatalakeDatabaseBackupCouldNotStartEvent.java

```diff
@@ -0,0 +1,33 @@
+package com.sequenceiq.datalake.flow.dr.event;
+
+import com.sequenceiq.datalake.flow.SdxEvent;
+
+public class DatalakeDatabaseBackupCouldNotStartEvent extends SdxEvent {
+    private final Exception exception;
+
+    public DatalakeDatabaseBackupCouldNotStartEvent(Long sdxId, String userId, Exception exception) {
+        super(sdxId, userId);
+        this.exception = exception;
+    }
+
+    public static DatalakeDatabaseBackupCouldNotStartEvent from(SdxEvent event, Exception exception) {
+        return new DatalakeDatabaseBackupCouldNotStartEvent(event.getResourceId(), event.getUserId(), exception);
+    }
+
+    @Override
+    public String selector() {
+        return "DatalakeDatabaseBackupCouldNotStartEvent";
```

Comment:
I won't go through all of your request, but please check them all.
The parent class already has defined this with `EventSelectorUtil` so you should depend on that instead of overriding it with the same

Reply:
will do